finalproject/designreview1.md

```diff
@@ -1,23 +1,56 @@
 # Design Review 1
 
-###This will take place in class on Friday, April 1
-
-Details will be posted here soon
-
-<!-- You will have **8 minutes** to present the following information:
-
-- Elevator pitch - A brief (30-45 second) pitch of your project idea.
-What is it? Why is it exciting?
-
-- In-depth project description including:
-  - system and infrastructure architecture diagrams.
-  - UI Mockups (sketches/wireframes)
-
-- Project roadmap with big picture features and a rough timeline -
-In what order will you tackle things? How long do you expect each aspect to take?
-
-- What are you the most worried about - The big unknowns, assumptions, showstoppers
-in your project. What have you done so far to understand these better, what are your next steps?
-
-There will be a few minutes at the end for questions and to collect feedback from the class and teaching team.
- -->
\ No newline at end of file
+### Format
+We’ll give you 10 min at the beginning of class to prepare, then the reviews will start at 3:30 in AC328 and three other classrooms in the AC. You’ll present to 1-2 other teams and 1-2 teaching team members, and you’ll have 15 minutes to do whatever will be most helpful for you moving forward -- get feedback or advice, do a codesign, do a code review... up to you. The flexibility we’re giving you means that we trust you to use the time productively.
+
+### Goals
+Here’s what we hope the design review will do for the class:
+- Facilitate inter-team communication -- if another team is doing something similar to what you’re doing, we want you to have the opportunity to help and learn from each other.
+- Act as a check-in so you have incentive to get moving on your project right away
+- Encourage you to spend time up front thinking about longer-term architecture decisions
+
+### Content
+We’d like you to start out by explaining your idea so the other people in the room (another team or two and 1-2 teachers/ninjas) have context for what you’re doing.
+
+Beyond that, you’re free to use the time however you think is best.
```

Comment:
The "Evidence you've done significant work so far" section below seems to imply that this should include an informative progress update on the work accomplished up to this point. Perhaps worth noting here?